src/System.Net.Utilities/src/System/Net/NetworkInformation/Icmpv6MessageConstants.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.NetworkInformation
+{
+    internal static class Icmpv6MessageConstants
```

Comment:
Same nit/concerns as v4: Capitalize the V in the name here, and make the const int blocks be enums.